mobile/src/main/java/ch/epfl/sdp/mobile/ui/social/ChessMatch.kt

```diff
@@ -1,6 +1,11 @@
 package ch.epfl.sdp.mobile.ui.social
 
-class ChessMatch(val adversary: String, val matchResult: MatchResult, val numberOfMoves: Int)
+class ChessMatch(
+    val uid: String,

```

Comment:
A `ChessMatch` should not have a `uid`, but instead you should use the same pattern as the generic `Person` (https://github.com/epfl-SDP/android/blob/main/mobile/src/main/java/ch/epfl/sdp/mobile/ui/social/SocialScreenState.kt#L16-L56) or the generic `Piece` (https://github.com/epfl-SDP/android/blob/main/mobile/src/main/java/ch/epfl/sdp/mobile/ui/game/ChessBoardState.kt#L20-L130).